source.js

```diff
@@ -10,9 +10,15 @@ function GET (path, cb) {
   // console.log(`### ${endpoint}${path}`);
   request({ url: `${endpoint}${path}`, json: true, timeout }, (err, res, body) => {
     if (err || !res || res.statusCode >= 400) {
-      let statusCode = res ? res.statusCode : 0
-        , statusMessage = res ? res.statusMessage : 'Unspecified error (likely a timeout)'
-      ;
+      let statusCode = res ? res.statusCode : 0;
```

Comment:
Salut Yannick!

This is a good feature and I will definitely take it, but do you mind aligning with the coding style first? Thanks.